src/packages.lisp

```diff
@@ -1,5 +1,12 @@
 (defpackage #:cl-raster/scene
-  (:use #:cl))
+  (:use #:cl)
+  (:export #:camera-width
+           #:camera-height
+           #:camera-center
+           #:camera-direction
+           #:camera-x-vector
+           #:camera-y-vector
+           #:scene-triangles))
```

Comment:
You should export all available API - some accessors are missing, also constructors (`make-camera`, `make-scene` and `make-light-source`) and type predicates (`camera-p`, `scene-p` and `light-source-p`)